exec/jdbc/src/test/java/org/apache/drill/jdbc/StatementTest.java

```diff
@@ -61,55 +71,129 @@ public static void tearDownStatement() throws SQLException {
   //////////
   // getQueryTimeout():
 
-  /** Tests that getQueryTimeout() indicates no timeout set. */
+  /**
+   * Test for reading of default query timeout
+   */
   @Test
-  public void testGetQueryTimeoutSaysNoTimeout() throws SQLException {
-    assertThat( statement.getQueryTimeout(), equalTo( 0 ) );
+  public void testDefaultGetQueryTimeout() throws SQLException {
+    Statement stmt = connection.createStatement();
+    int timeoutValue = stmt.getQueryTimeout();
+    assert( 0 == timeoutValue );
```

Comment:
I personally prefer the old construct using hamcrest. This `assert` is actually less useful for two reasons:
- there's no error message (hamcrest prints a nice error message based on the arguments)
- if the assert fails, you don't know `timeoutValue`'s value

Reply:
+1